evap/evaluation/tests/test_commands.py

```diff
@@ -336,7 +336,7 @@ def test_dont_remind_already_voted(self):
         self.assertEqual(mock.call_count, 0)
         self.assertEqual(len(mail.outbox), 0)
 
-    @override_settings(TEXTANSWER_REVIEW_REMINDER_WEEKDAYS=list(range(0, 8)))
+    @override_settings(TEXTANSWER_REVIEW_REMINDER_WEEKDAYS=list(range(8)))
```

Comment:
Any idea why we have the words "weekdays" and "8" in this combination here?

Reply:
should this be changed to `range(7)`? The comment in the settings indicate this should be `range(7)`:
https://github.com/e-valuation/EvaP/blob/main/evap/settings.py#L64-L67
```suggestion
    @override_settings(TEXTANSWER_REVIEW_REMINDER_WEEKDAYS=list(range(7)))
```